scripts/update_version.py

```diff
@@ -17,7 +17,11 @@ def update_version():
     Opens file and updates the version number
     """
     release_version = os.getenv("VERSION")[1:]
-    last_day_of_month = date.today() + relativedelta(day=31)
+    release_date = (
+        date.today()
+        if "rc" in release_version
+        else date.today() + relativedelta(day=31)
+    )
```

Comment:
The final release will be at the end of the month, but the rc releases will be in between.

Reply:
Not all months have 31 days

Reply:
This always returns the last day of the month. The current workflows use the same line and it can be seen that the headings in the CHANGELOG are correct for months that don't have 31 days.